hook-janitor/src/config.rs

```diff
@@ -44,7 +44,7 @@ pub struct KafkaConfig {
     #[envconfig(default = "false")]
     pub kafka_tls: bool,
 
-    #[envconfig(default = "app_metrics")]
+    #[envconfig(default = "clickhouse_app_metrics")]
```

Comment:
Need to double-check we're not also setting it in the chart. let's remove from the chart if that's the case, we probably won't need to update

Reply:
Good catch, it's in the chart.